components/molecules/checkbox/index.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import classNames from 'classnames'
+
+import Input from '_components/atoms/input'
+
+import styles from './styles.css'
+
+const Checkbox = props => {
+  const { className, isChecked, disabled } = props
+  const inputClasses = classNames({
+    [className]: true,
+    [styles.input]: true,
+    [styles.checked]: isChecked,
+    [styles.disabled]: disabled,
+  })
+
+  return (
+    <label className={styles.checkbox} role="checkbox" aria-checked={isChecked}>
+      <Input {...props} type="checkbox" className={inputClasses} checked={isChecked} />
+    </label>
+  )
+}
+
+Checkbox.propTypes = {
+  className: PropTypes.string,
+  id: PropTypes.string.isRequired,
+  isChecked: PropTypes.bool.isRequired,
```

Comment:
In this line you put that property is required and on the `defaultProps` you attributed a value for this.